python/paddle/fluid/tests/unittests/test_traced_layer_err_msg.py

```diff
@@ -174,5 +180,25 @@ def test_linear_net_with_none(self):
                                                                         [in_x])
 
 
+class TestTracedLayerSaveInferenceModel(unittest.TestCase):
+    """test save_inference_model will automaticlly create non-exist dir"""
+
+    def setUp(self):
+        self.save_path = "./nonexist_dir/fc"
+        import shutil
+        if os.path.exists(os.path.dirname(self.save_path)):
+            shutil.rmtree(os.path.dirname(self.save_path))
+
+    def test_mkdir_when_input_path_non_exist(self):
+        fc_layer = SimpleFCLayer(3, 4, 2)
+        input_var = paddle.to_tensor(np.random.random([4, 3]).astype('float32'))
+        with fluid.dygraph.guard():
+            dygraph_out, traced_layer = fluid.dygraph.TracedLayer.trace(
```

Comment:
We should write new test cases for Paddle 2.0 path. So `fluid.dygraph.TracedLayer` should be changed to `paddle.jit.TracedLayer`. However, since it is not so important and this PR needs many approves. I will approve it and you can change in next PR.

Reply:
Thank you for your suggestion，I will change these two in next pr.